ui/dashboards/src/context/DashboardProvider/panel-editor-slice.ts

```diff
@@ -195,7 +195,7 @@ export function createPanelEditorSlice(): StateCreator<
         },
         applyChanges: (next) => {
           const panelDef = createPanelDefinitionFromEditorValues(next);
-          const panelKey = removeWhiteSpacesAndSpecialCharacters(next.name);
+          const panelKey = createUniquePanelKey(next.name);
```

Comment:
Just a suggestion (feel free to ignore), but I really like the fact that the panel keys don't have numbers by default under the old implementation. Would be really nice to only add a number if a given panel key is already taken (and then _ideally_ I think you'd add whatever the next number was starting from `2` and counting up that gets you a unique key, but maybe that's overkill).

Anyway, just a thought that I think leads to nicer/easier to read JSON.

Reply:
good point, i'll change it since the number from `generateId` will look random